packages/kit/src/views/Prime/components/PrimeLoginPasswordDialog/PrimeLoginPasswordDialog.tsx

```
@@ -17,20 +18,46 @@
+          }),
+        };
       case 4:
-        return { width: '100%', color: '$bgSuccessStrong', text: 'Strong' };
+        return {
+          width: '100%',
+          color: '$bgSuccessStrong',
+          text: intl.formatMessage({
+            id: ETranslations.prime_password_level_strong,
+          }),
+        };
       case 0:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)